sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_storage_utils.py

```diff
@@ -191,10 +191,7 @@ def get_ds_name_and_path_prefix(asset_uri: str, registry_name: Optional[str] = N
     if registry_name:
         try:
             split_paths = re.findall(STORAGE_URI_REGEX, asset_uri)
-            if split_paths[0][3] == "":
-                path_prefix = split_paths[0][2]
```

Comment:
if you want the last index only?
we can try this split_paths[0][-1], may be more future proof??